src/poliastro/tests/test_czml.py

```diff
@@ -645,3 +645,45 @@ def test_czml_invalid_orbit_epoch_error():
         "ValueError: The orbit's epoch cannot exceed the constructor's ending epoch"
         in excinfo.exconly()
     )
+
+
+@pytest.mark.skipif("czml3" not in sys.modules, reason="requires czml3")
+def test_czml_timezone():
```

Comment:
Isn't this test a bit redundant since it's guaranteed that the timezone format will be tested in every other test anyway?

Reply:
Yes. I also think that the test is redundant. Should I delete this test?

Reply:
I think it would be better if you did since it doesn't test for any specific edge-case or other functionality

Reply:
I have removed the test. Please tell if there is anything else to update.

Reply:
Looks good! Could you perhaps squash and rebase?

Reply:
I have tried to squash the commits and rebase to master. Please check if I did it correctly.